core/conn/odb/src/odb.c

```diff
@@ -9889,7 +9889,9 @@ static void Oextract(int eid)
             if ( etab[eid].fho ) {
                 (*hdfswrite)(hfs, etab[eid].fho, (void *)xbuff, xbuffl);
             } else {
-                (void)fwrite ( xbuff, 1, xbuffl, etab[eid].fo );
+                if ( etab[eid].fo == stdout ) {
```

Comment:
If the current problem is that  etab[eid].fo was NULL and we tried to write to it, will it not be better if the change was 
if ( etab[eid].fo ) {
fwrite (...)
}
This way we can handle cases where etab[eid].fo is a generic file handle too.
Maybe the concern is on Line 9882 we have already called fclose on this file handle   (if its not stdout) ?
This change is safe. You could think about whether it could be made more general.